Plugins/Public/dronebays/Utility.cpp

```diff
@@ -0,0 +1,343 @@
+#include "Main.h"
+
+void Utility::SetRepNeutral(uint clientObj, uint targetObj)
+{
+	int targetRep;
+	int droneRep;
+	pub::SpaceObj::GetRep(targetObj, targetRep);
+	pub::SpaceObj::GetRep(clientObj, droneRep);

```

Comment:
are we going to run into problems with multiple drones having the same affiliation?